spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -121,11 +121,11 @@ def compute_presence_ratios(waveform_extractor, bin_duration_s=60.0, mean_fr_rat
     num_bin_edges = total_length // bin_duration_samples + 1
     bin_edges = np.arange(num_bin_edges) * bin_duration_samples
 
-    mean_fr_ratio_thres = float(mean_fr_ratio_thres)
-    if mean_fr_ratio_thres < 0:
+    mean_fr_ratio_thresh = float(mean_fr_ratio_thresh)
+    if mean_fr_ratio_thresh < 0:
```

Comment:
Should we have an upper bound too? At least `0 <= mean_fr_ratio_thresh <= 1`

Reply:
Yes why not, I didn't put it in because it's not logically required and I thought it might be useful in some case to set a high threshold (eg to catch units with very asymmetrical firing). But again there's probably another much more mathematically sound way of setting a threshold... As you see fit. Maybe a warning if >1?